readme.md

```diff
@@ -18,7 +18,7 @@ Avoid the need to enter your remote user password for SSH by setting up an
 
 ## How to use
 
-Typical usage for users who have done minimal and maximal set up are
```

Comment:
I think this one was actually correct as is, "set up" is a verb here.
Happy to be corrected though

Reply:
I corrected it into "set-ups" so that the sentence is grammatically consistent and feels natural to read. Now that I am reading this again, I actually found that we need the plural "usages" here.

> Typical usages for users [who have done {noun}/{verb-ing}] are ...

If we break down the sentence grammatically like above, then it makes more sense that whatever follows "done" before "are" should be a noun (with adjectives).

Maybe there is another way of interpreting this. Either way, I think the meanings are delivered successfully, so we could just keep it as-is.

Reply:
I agree 100% on "usages", thanks for the catch.
The noun vs verb is a bit trickier, but I'm now inclined to agree with you that `set up/setup/set-up` is a noun.